src/consts.rs

```diff
@@ -8,6 +8,7 @@ pub const PREFIX_FILE_NAME: &str = "prefix";
 pub const ENVIRONMENTS_DIR: &str = "envs";
 pub const SOLVE_GROUP_ENVIRONMENTS_DIR: &str = "solve-group-envs";
 pub const PYPI_DEPENDENCIES: &str = "pypi-dependencies";
+pub const TASK_CACHE_DIR: &str = "task-cache";
```

Comment:
How would you feel about changing this to `task-cache-v0` so we can change the format of the files a little easier in the future?

Reply:
Done!